book_config.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python2
 # vim:fileencoding=UTF-8:ts=4:sw=4:sta:et:sts=4:ai
+'''Creates dialog to allow finer control of book specific settings'''
```

Comment:
Comments on comments ftw: "finer" can be removed.  This is the only way to control book-specific settings, everything else would be a global-setting.